launch.go

```diff
@@ -268,6 +268,11 @@ func createEnvironment(base map[string]string, secrets screwdriver.Secrets) []st
 	for k, v := range combined {
 		envStrings = append(envStrings, strings.Join([]string{k, v}, "="))
 	}
+
+	for k, v := range build.Environment {
```

Comment:
Right, shouldn't we be adding our SD environment variables in the config-parser?  That way we can control which order they apply in and what they are.